packages/feedback/package.json

```diff
@@ -37,7 +37,7 @@
     "lint:prettier": "prettier --check \"{src,test,scripts}/**/*.ts\"",
     "test": "jest",
     "test:watch": "jest --watch",
-    "yalc:publish": "ts-node ../../scripts/prepack.ts --bundles && yalc publish ./build/npm --push"
+    "yalc:publish": "ts-node ../../scripts/prepack.ts --bundles && yalc publish ./build/npm --push --sig"
```

Comment:
We should apply this to all packages? This updates the bundle signature so that e.g. webpack-dev-server's watch mode will work. Otherwise the package gets cached due to version of the package not changing.

Reply:
sounds reasonable to me!